ivc/src/ivc/columns.rs

```diff
@@ -245,7 +252,13 @@ pub enum IVCColumn {
     Block1InputRepacked150(usize),
 
     /// 1 hash per row
-    Block2Hash(PoseidonColumn<IVC_POSEIDON_STATE_SIZE, IVC_POSEIDON_NB_FULL_ROUND>),
+    Block2Hash(
```

Comment:
Do I understand correctly that the fixed selector changes are not yet in master?

Reply:
It is in master. Why makes you think it is not?